### DISCUSSION TITLE
Can I change the name that appears when using the /say command?

### DISCUSSION THREAD

**Discussion by Babineauxx** [Tiiffi/mcrcon#84] [Q&A]
I would like to change the name that appears when using the /say command.

Currently I have been using /tellraw for messages generated by my systemd service but would like to talk to my users via console sometimes. I find myself explaining who rcon is quite often.

Can i change this 'username'?

**Comment by Tiiffi** [comment]
Hi,

Unfortunately it's hardcoded in server-side.

Maybe as a workaround just add nickname in the begginning of the line.
Something like `[nick] message` so the output would look like `[Rcon][Nick] message`.

No the prettiest solution but at least something.
